default.xml

```diff
@@ -6,60 +6,61 @@
 
   <!-- remotes -->
   <remote name="jhnc-oss" fetch="https://github.com/jhnc-oss" />
+  <remote name="meta-qt5" fetch="https://github.com/meta-qt5" />
 
   <!-- layers -->
   <project remote="jhnc-oss"
            name="protos"
            path="protos"
-           revision="main" />
+           revision="kirkstone" />
 
   <project remote="jhnc-oss"
            name="meta-protos"
            path="meta-protos"
-           revision="main" />
+           revision="kirkstone" />
```

Comment:
There's some clarification needed: Is `main == kirkstone` or is `kirkstone` a separate branch?

Reply:
As of now  kirkstone is a separate branch which will be in sync with main as long as we don'ts switch to next LTS candidate.